core/internal/runfilestest/runfilestest.go

```diff
@@ -0,0 +1,21 @@
+package runfilestest
+
+import (
+	"github.com/wandb/wandb/core/internal/runfiles"
+	"github.com/wandb/wandb/core/internal/settings"
+	"github.com/wandb/wandb/core/pkg/observability"
+	"github.com/wandb/wandb/core/pkg/service"
+)
+
+// Sets nice default parameters for testing purposes.
```

Comment:
we should generalize it, this is really convenient

Reply:
it's a pattern I like a lot for tests that use real objects rather than mocks